org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel.releng/tp/domainmodel.target

```diff
@@ -11,10 +11,10 @@
       <repository location="https://download.eclipse.org/releases/2019-06"/>
     </location>
     <location includeMode="slicer" includeAllPlatforms="true" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
-      <unit id="org.eclipse.emf.mwe.sdk.feature.group" version="1.4.0"/>
-      <unit id="org.eclipse.emf.mwe2.runtime.sdk.feature.group" version="2.10.0"/>
-      <unit id="org.eclipse.emf.mwe2.language.sdk.feature.group" version="2.10.0"/>
-      <repository location="https://download.eclipse.org/modeling/emft/mwe/updates/releases/2.10.0"/>
+      <unit id="org.eclipse.emf.mwe.sdk.feature.group" version="1.5.0.201906111547"/>
```

Comment:
Should we use omni versions here?

Reply:
there is this strange tpd file and @kthoms tool creates this here (instead of 0.0.0)
@kthoms what do you think? should we ditch tpd?